resources/views/page-builder/partners.blade.php

```diff
@@ -1,31 +1,24 @@
-<div class="mx-auto mb-12 flex max-w-5xl flex-col items-center px-6 md:mb-32"{{ ($component_id ?? false) ? " id=" . $component_id : "" }}>
-    @if(($title_heading ?? false) || ($title_subheading ?? false))
-        <h2 class="text-3xl font-semibold md:text-4xl">
-            @if($title_heading ?? false)
-                <span class="text-heading">{{ $title_heading }}</span>
-            @endif
-            @if($title_subheading ?? false)
-                <span class="text-secondary-100">
-                    {{ $title_subheading }}
-                </span>
-            @endif
-        </h2>
+<div id="{{ $component_id }}" class="component mx-auto flex max-w-5xl flex-col items-center px-6">
+    @if ($title_heading)
+        <div class="text-3xl font-semibold text-heading prose-strong:text-secondary-100 md:text-4xl">
+            {!! $title_heading !!}
+        </div>
     @endif
-    @if($text ?? false)
+    @if ($text)
         <div class="mt-6 max-w-md text-center text-sm text-heading">
             {!! $text !!}
         </div>
     @endif
-    @if($columns ?? false)
+    @if ($columns)
         <div class="relative mt-12 flex w-full flex-wrap items-start justify-center overflow-hidden pb-5">
-            @foreach($columns as $column)
+            @foreach ($columns as $column)
                 <x-package.group :checked="$loop->first">
-                    @if($column->title ?? false)
-                        <x-slot name="label" class="{{ $loop->first ? 'rounded-l-lg' :  ($loop->last ? 'rounded-r-lg' : '')}}">{{ $column->title }}</x-slot>
+                    @if ($column->title)
+                        <x-slot name="label" class="{{ $loop->first ? 'rounded-l-lg' : ($loop->last ? 'rounded-r-lg' : '') }}">{{ $column->title }}</x-slot>
```

Comment:
Do you need the `$loop->first` and `$loop->last`? You can't use `first: & last:`? https://tailwindcss.com/docs/hover-focus-and-other-states#first-last-odd-and-even